### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by DianaBeki**
### I was able to do the following
Use React documentation.
Use React components.
Use React props.
Use React Router.
Connect React and Redux.
Handle events in a React app.
Write integration tests with a React testing library.
Use styles in a React app.
Use React life cycle methods.
Apply React best practices and language style guides in code.
Use store, actions and reducers in React.

### CODE REVIEW

**Comment by bellom** [comment]
## Hi @DianaBeki  👏  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

**CRR marked as invalid because of the following reasons stated below: 👇** 

**Kindly fix the linters error.**

Heads-up: It would be nice to add a unit test as stated in the [project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Testing%20requirements) to avoid a re-review.

Cheers and Happy coding!👏👏👏

**Comment by DianaBeki** [comment]
Hello @bellom i have made all the required changes including linters. Kindly check my latest code I will appreciate

File: README.md [hunk 1]
```diff
@@ -1 +1,185 @@
...
+- Can browse next pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](coming soon)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (ahmetbozaci) [inline README.md:72]: According to the requirements kindly implement these;
- [ ] Deploy your website and put the link in the [readme](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=The%20project%20should,your%20README%20file.).
- [ ] Create a presentation video. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Once%20you%20have,a%20small%20picture.)

File: src/__tests__/MealDetails.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable */
```
Comment (ahmetbozaci) [inline src/__tests__/MealDetails.test.js:1]: OPTIONAL
- [ ] Kindly dont disable linters. Here is the [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=Do%20not%20disable%20Eslint%20or%20Stylint%20using%20inline%20comments%20within%20your%20code%20if%20not%20specified%20in%20the%20project%20requirement.)

File: src/__tests__/MealList.test.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+import axios from 'axios';
```
Comment (ahmetbozaci) [inline src/__tests__/MealList.test.js:2]: OPTIONAL
- [ ] Kindly dont disable linters. Here is the [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=Do%20not%20disable%20Eslint%20or%20Stylint%20using%20inline%20comments%20within%20your%20code%20if%20not%20specified%20in%20the%20project%20requirement.)

File: src/__tests__/mealSlice.test.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable */
+import mealReducer, { fetchMeal } from '../redux/mealSlice';
```
Comment (ahmetbozaci) [inline src/__tests__/mealSlice.test.js:2]: OPTIONAL
- [ ] Kindly dont disable linters. Here is the [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#:~:text=Do%20not%20disable%20Eslint%20or%20Stylint%20using%20inline%20comments%20within%20your%20code%20if%20not%20specified%20in%20the%20project%20requirement.)

File: README.md [hunk 5]
```diff
@@ -1 +1,185 @@
-# meal_point
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="readme-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```
Comment (ahmetbozaci) [inline README.md:6]: OPTIONAL
- [ ] If not have logo kindly delete it because it is broken now.

File: src/__tests__/MealList.test.js [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+    await store.dispatch(fetchMeal());
+
+    const actionTypes = store.getActions().map((action) => action.type);
+    expect(actionTypes).toEqual(expectedActions);
+  });
+});
+
+
+
```
Comment (ahmetbozaci) [inline src/__tests__/MealList.test.js:33]: - [ ] Tests are not passed kindly fix it and test it before next review.
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️

 Hello @DianaBeki ✋🏼 

_This is a manual review_


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._


___



[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**